app/src/main/java/org/transdroid/search/TorrentSite.java

```diff
@@ -115,6 +117,18 @@ public ISearchAdapter getAdapter() {
 			return new NcoreAdapter();
 		}
 	},
+	Nebulance {
```

Comment:
This is sorted alphabetically so MoreThanTv should be before Nebulance.

Reply:
Done